src/pages/MainPage/MainPage.js

```diff
@@ -1,9 +1,229 @@
 import React, { Component } from 'react';
+import MainItem from './MainItem/MainItem';
 import './MainPage.scss';
 
 class MainPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      eventCount: 1,
+      insideCount: 1,
+      list: [],
+    };
+    this.saleMargin = 0;
+    this.insideMargin = 0;
+  }
+
+  handleNextClick = () => {
+    const { eventCount, list } = this.state;
+    const length = Number(list.map(item => item.eventList.length));
+    console.log(length);
+
+    if (eventCount < length) {
+      this.setState({
+        eventCount: eventCount + 1,
+      });
+      this.saleMargin -= 421;
+    }
+  };
+
+  handlePrevClick = () => {
+    const { eventCount, list } = this.state;
+    const length = Number(list.map(item => item.eventList.length));
+
+    if (!eventCount < length && eventCount > 1) {
+      this.setState({
+        eventCount: eventCount - 1,
+      });
+      this.saleMargin += 421;
+    }
+  };
+
+  handleInSideNextClick = () => {
+    const { insideCount, list } = this.state;
+    const length = Number(list.map(item => item.insideList.length));
+
+    if (insideCount < length) {
+      this.setState({
+        insideCount: insideCount + 1,
+      });
+
+      this.insideMargin -= 1562;
+    }
+  };
+
+  handleInSidePrevClick = () => {
+    const { insideCount, list } = this.state;
+    const length = Number(list.map(item => item.insideList.length));
+
+    if (!insideCount < length && insideCount > 1) {
+      this.setState({
+        insideCount: insideCount - 1,
+      });
+
+      this.insideMargin += 1562;
+    }
+  };
+
+  componentDidMount() {
+    fetch('http://localhost:3007/data/mainData-songhyun.json')
```

Comment:
```suggestion
    fetch('/data/mainData-songhyun.json')
```
localhost:3007 등의 현재 구동되고 있는 서버의 주소는 안적어주셔도 됩니다. 절대경로로 적으면 CRA 내부적으로 자동으로 현재 구동되고 있는 서버 주소를 넣어줍니다, 오히려 이렇게 임의적으로 기입하게 되면 3007포트가 아닌 3000포트 등에서 돌아갈 때는 에러가 발생하겠네요